### DISCUSSION TITLE
problem in the bonsai, the program closes the windows when analyzing videos

### DISCUSSION THREAD

**Discussion by GilMS11** [bonsai-rx/bonsai#1764] [Q&A]
I installed bonsay on a macbook through a UTM on which I use windows 11. I was able to install bonsay and set up the layout to analyze my videos, however, when I select the video I want to analyze, the video window closes and the program does too and it doesn't I can complete the analysis. How do I solve this problem?

**Comment by GilMS11** [comment]
I can't open the FileCapture and roiactivity

**Comment by bruno-f-cruz** [comment]
Please share the workflow you are trying to use. Thanks
